test/system/notify-liquidated.test.js

```diff
@@ -0,0 +1,165 @@
+  // deposit lot size is 5 BTC
+  const lotSize = to1e18(5)
+
+  let tbtcToken
+  let underwriterToken
+  let assetPool
+  let signerBondsSwapStrategy
+  let coveragePool
+  let riskManagerV1
+  let tbtcDeposit
+
+  let governance
+  let rewardsManager
+  let bidder
+  let bidder1
+
+  before(async () => {
+    await resetFork(startingBlock)
+    governance = await ethers.getSigner(0)
+    rewardsManager = await ethers.getSigner(1)
+
+    tbtcToken = await ethers.getContractAt("IERC20", tbtcTokenAddress)
+
+    const UnderwriterToken = await ethers.getContractFactory("UnderwriterToken")
+    underwriterToken = await UnderwriterToken.deploy("Coverage KEEP", "covKEEP")
+    await underwriterToken.deployed()
+
+    const AssetPool = await ethers.getContractFactory("AssetPool")
+    assetPool = await AssetPool.deploy(
+      keepTokenAddress,
+      underwriterToken.address,
+      rewardsManager.address
+    )
+    await assetPool.deployed()
+    await underwriterToken.transferOwnership(assetPool.address)
+
+    const SignerBondsSwapStrategy = await ethers.getContractFactory(
+      "SignerBondsEscrow"
+    )
+    signerBondsSwapStrategy = await SignerBondsSwapStrategy.deploy()
+    await signerBondsSwapStrategy.deployed()
+
+    const Auction = await ethers.getContractFactory("Auction")
+
+    const masterAuction = await Auction.deploy()
+    await masterAuction.deployed()
+
+    const CoveragePool = await ethers.getContractFactory("CoveragePool")
+    coveragePool = await CoveragePool.deploy(assetPool.address)
+    await coveragePool.deployed()
+    await assetPool.transferOwnership(coveragePool.address)
+
+    const RiskManagerV1 = await ethers.getContractFactory("RiskManagerV1")
+    riskManagerV1 = await RiskManagerV1.deploy(
+      tbtcToken.address,
+      tbtcDepositTokenAddress,
+      coveragePool.address,
+      signerBondsSwapStrategy.address,
+      masterAuction.address,
+      auctionLength,
+      bondAuctionThreshold
+    )
+    await riskManagerV1.deployed()
+
+    await coveragePool
+      .connect(governance)
+      .approveFirstRiskManager(riskManagerV1.address)
+
+    tbtcDeposit = await ethers.getContractAt("IDeposit", depositAddress)
+
+    bidder = await impersonateAccount(bidderAddress)
+    bidder1 = await impersonateAccount(bidderAddress1)
+  })
+
+  describe("when notified on liquidated auction", () => {
+    let auction
+
+    before(async () => {
+      await tbtcDeposit.notifyRedemptionSignatureTimedOut()
+
+      // The deposit's auction must offer at least 75% of bonds to be accepted
+      // by the risk manager. At starting block, the deposit's auction exposes
+      // 66% we need an additional 9% to pass the risk manager threshold. To get
+      // this part, we need 22870 seconds to elapse. This is because the auction
+      // length is 86400 seconds (24h) and there is 34% of bonds remaining.
+      // So, additional 9% will be offered after 9/34 * 86400s.
+      await increaseTime(22870)
```

Comment:
Please take a look: https://github.com/keep-network/coverage-pools/pull/84#discussion_r653381960

Reply:
Agreed, and I suggest we us 66%, more info [here](https://github.com/keep-network/coverage-pools/pull/84#discussion_r654331791).

Reply:
Sure, done